### PR TITLE
Fix issue with order of flags

### PR DESCRIPTION
**Pull request by umohnani8**
The order of the flags was casuing issue.
Enabled SkipArgReorder to fix the problem.

Signed-off-by: umohnani8 <umohnani@redhat.com>

### CODE REVIEW

**Comment by umohnani8** [comment]
@rhatdan @baude PTAL

**Comment by rhatdan** [comment]
This looks fine to me, but is there an issue or an explanation of what the problem you were seeing?

**Comment by baude** [comment]
I would like to see a test that will prevent regression.

**Comment by umohnani8** [comment]
@rhatdan the issue was raised by QE in https://trello.com/c/WHprWjvV/744-1-cri-okpod-add-dir-transport-to-kpod-save.
This is what was happening:
```
 ➜  libpod git:(stats) sudo podman save --compress --format oci-dir -o blah alpine     
option --format requires a value

➜  libpod git:(stats) sudo podman save --format oci-dir --compress -o blah alpine
Getting image source signatures
Copying blob sha256:04a094fe844e055828cb2d64ead6bd3eb4257e7c7b5d1e2af0da89fa20472cf4
4.20 MB / 4.20 MB [========================================================] 0s
Copying config sha256:501d1a8f0487e93128df34ea349795bc324d5e0c0d5112e08386a9dfaff620be
584 B / 584 B [============================================================] 0s
Writing manifest to image destination
Storing signatures
```

**Comment by umohnani8** [comment]
Added test, it passes for now but will fail if a regression occurs.

File: test/podman_save.bats [hunk 1]
```diff
@@ -43,16 +43,23 @@ function setup() {
...
-@test "podman save to directory wit v2s2 (docker) format" {
+@test "podman save to directory with v2s2 (docker) format" {
 	run ${PODMAN_BINARY} ${PODMAN_OPTIONS} save --format docker-dir -o alp-dir $ALPINE
 	echo "$output"
 	[ "$status" -eq 0 ]
 	rm -rf alp-dir
 }
+
+@test "podman save to directory with compression" {
+	run ${PODMAN_BINARY} ${PODMAN_OPTIONS} save --compress --format docker-dir -o alp-dir $ALPINE
```
Comment (rhatdan) [inline test/podman_save.bats]: Can you add both forms

```
+	run ${PODMAN_BINARY} ${PODMAN_OPTIONS} save --compress --format=docker-dir -o alp-dir $ALPINE
￼```

Comment (umohnani8) [reply test/podman_save.bats]: Fixed

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**